.binstar.yml

```diff
@@ -1,15 +1,13 @@
 package: poliastro
 install:
-  - conda config --set always_yes true
+  - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels poliastro
 script:
-  # Force conda build to use specific versions of Python and Numpy
-  - export CONDA_PY=$(python -c "from __future__ import print_function;from sys import version_info as ver;print(str(ver.major) + str(ver.minor))")
-  - export CONDA_NPY=$(python -c "from __future__ import print_function;import numpy;npver=numpy.__version__;print(npver[:npver.rfind('.')].replace('.',''))")
-  # Build once the env variables are set
-  - conda build buildscripts/condarecipe
```

Comment:
In answer to your question I can tell you, in my experience dealing with this hell, that this will have no effect; if no strict dependency on **NumPy** is declared it fallbacks to the default behavior and upgrades the library when setting up the building environment. In fact, since the building process doesn't inherit environment variables by default (in order to be completely reproducible) it's truly difficult to establish mechanisms to prevent the upgrade.

About your suggestion, yes it'll result in a cleaner script, I'll take care of it. However, it's a pity that no environments variables based on the engine configuration are set; this would have made all the process more straightforward.